cmd/resources/xrootd.cfg

```diff
@@ -14,43 +14,43 @@
 # limitations under the License.
 #
 
-all.manager {{.ManagerHost}}+ {{.ManagerPort}}
+all.manager {{.Xrootd.ManagerHost}}+ {{.Xrootd.ManagerPort}}
 all.role server
 if exec xrootd
-  xrd.port {{.Port}}
-  xrd.protocol http:{{.Port}} libXrdHttp.so
+  xrd.port {{.Xrootd.Port}}
+  xrd.protocol http:{{.Xrootd.Port}} libXrdHttp.so
 fi
-xrd.tls {{.TLSCertificate}} {{.TLSKey}}
-{{if .TLSCertDir}}
-xrd.tlsca certdir {{.TLSCertDir}}
+xrd.tls {{.Server.TLSCertificate}} {{.Server.TLSKey}}
+{{if .Xrootd.TLSCertDir}}
+xrd.tlsca certdir {{.Xrootd.TLSCertDir}}
```

Comment:
Can you rename this one `Server.TLSCACertificateDirectory`?  It's not XRootD-specific, really -- just only used by XRootD (currently!).